scripts/automation/create_release_tag.py

```diff
@@ -0,0 +1,446 @@
+    try:
+        with open(TARGET_FILE, 'r', encoding='utf-8') as f:
+            data = json.load(f)
+
+        for ext_name, versions in data["extensions"].items():
+            for version in versions:
+                if version["sha256Digest"] == sha:
+                    return version["filename"], version["downloadUrl"]
+
+        return None, None
+
+    except FileNotFoundError:
+        print(f"Error: {TARGET_FILE} not found")
+        raise
+    except json.JSONDecodeError:
+        print(f"Error: Invalid JSON format in {TARGET_FILE}")
+        raise
+
+
+def get_release_info(tag_name: str) -> Tuple[Optional[int], Optional[int]]:
+    try:
+        url = f"{base_url}/releases/tags/{tag_name}"
+
+        response = requests.get(url, headers=headers)
+        response.raise_for_status()
+
+        data = response.json()
+
+        release_id = data.get("id")
+        release_body = data.get("body")
+        assets = data.get("assets", [])
+        asset_id = assets[0].get("id") if assets else None
+
+        return release_id, release_body, asset_id
+    except requests.RequestException as e:
+        print(f"API request failed: {e}")
+        return None, None, None
+    except (KeyError, IndexError) as e:
+        print(f"Failed to parse response data: {e}")
+        return None, None, None
+
+
+def update_release_body(release_id: int, commit_sha: str, old_body: str, sha: str, tag_name: str) -> bool:
+    try:
+        url = f"{base_url}/releases/{release_id}"
+        ref_url = f"{base_url}/git/refs/tags/{tag_name}"
+        data = {
+            "sha": commit_sha,
+            "force": True
+        }
+        print(f"Updating tag {tag_name} to point to commit {commit_sha}")
+        response = requests.patch(ref_url, headers=headers, json=data)
+        response.raise_for_status()
+        print(f"Successfully updated tag {tag_name}")
+
+        sha_pattern = r'[a-fA-F0-9]{64}'
+        new_body = old_body
+
+        found_shas = re.finditer(sha_pattern, old_body)
+        for match in found_shas:
+            old_sha = match.group()
+
+            if old_sha != commit_sha:
+                new_body = new_body.replace(old_sha, sha)
+                break
+
+        payload = {
+            "body": new_body
+        }
+
+        response = requests.patch(url, json=payload, headers=headers)
+        response.raise_for_status()
+
+        return True
+    except requests.RequestException as e:
+        print(f"Failed to update release: {e}")
+        if hasattr(e.response, 'text'):
+            print(f"Response: {e.response.text}")
+        return False
+    except Exception as e:
+        print(f"Unexpected error: {e}")
+        return False
+
+
+def update_release_asset(wheel_url: str, asset_id: int, release_id: int) -> bool:
+    try:
+        print(f"Downloading wheel from {wheel_url}")
+        wheel_response = requests.get(wheel_url)
+        wheel_response.raise_for_status()
+
+        if asset_id is not None:
```

Comment:
It is not possible to directly overwrite the previous wheel package; I can only delete it first and then re-upload it.